redbot/core/app_commands/errors.py

```diff
@@ -0,0 +1,12 @@
+"""Errors module for the app_commands package."""
+from discord import app_commands
+
+__all__ = ("UserFeedbackCheckFailure",)
```

Comment:
This package will not be public, so `__all__` should not be used. This class name should remain in the `__all__` of `app_commands.__init__`, as it is only guaranteed to exist when imported directly from `redbot.core.app_commands`.

> Lack of `__all__` in the module means that all of its names are considered private APIs.

https://docs.discord.red/en/stable/version_guarantees.html#developer-guarantees